src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/NuGet/NuGetRestorer.cs

```diff
@@ -188,18 +155,6 @@ await _solutionRestoreService
             return Task.CompletedTask;
```

Comment:
Can we actually await in this method instead of running everything synchronously?

Reply:
Everything is sync in this method, it shouldn't be async.

Reply:
I'll remove the async portion of it.

Reply:
https://github.com/dotnet/project-system/pull/3099